README.md

```diff
@@ -83,3 +83,31 @@ Hash Generation
 For the main component (`metadata.component`), the plugin computes the SHA-256 hash of the release tarball (`<name>-<version>.tar.gz`) found in the release directory.
 
 If the tarball does not exist (e.g., because `rebar3 tar` hasn't been run), no hash is included for the main component, and a warning is logged.
+
+External References
+-------------------
+
+The plugin supports external references for components, which are automatically extracted from the `links` field in your `.app.src` file or from Hex metadata for dependencies.
+
+All standard CycloneDX external reference types are supported. Additionally, for convenience, the plugin supports common field names used by the Erlang/Elixir community, which are automatically mapped to their CycloneDX equivalents:
+
+- `"GitHub"` → `"vcs"`
+- `"Homepage"` → `"website"`
+- `"Changelog"` → `"release-notes"`
+- `"Sponsor"` → `"support"`
```

Comment:
Under "Sponsor" I would expect a list of people or organisations that sponsor the application, but under support I would expect link where I can find help. Like a forum or mail.